examples/aditof-demo/aditofdemocontroller.h

```diff
@@ -58,6 +58,8 @@ class AdiTofDemoController {
     aditof::Status enableNoiseReduction(bool en);
     aditof::Status setNoiseReductionThreshold(uint16_t threshold);
 
+    aditof::Status setCameraRevision(std::string revision);
```

Comment:
It is a good practice to pass argument by reference. Another good practice: argument should be const as the intention is not to modify it.

Reply:
"It is a good practice to pass argument by reference" I thought this applies only when the argument is big like a structure that occupies a lot of memory. I will change. thanks

Reply:
String can be one of those big structures

Reply:
changed, thanks

Reply:
nothing has changed. Looks the same.

Reply:
sorry, now there should be the right modification